addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php

```diff
@@ -115,6 +115,12 @@ function ($name) {
                     );
                 }
             ),
+	        new \Twig_SimpleFunction(
+		        'diffForHumans',
+		        function ($date) {
+			        return $date->diffForHumans();
```

Comment:
is it possible to use like

function ($date,$functionName) {
$date->{$functionName}();

Reply:
@muammertop` {{ "13/03/2020"|date("Y") }}`